src/v/cluster/topic_table.cc

```diff
@@ -1253,6 +1325,20 @@ ss::future<> topic_table::apply_snapshot(
     _last_applied_revision_id = snap_revision;
 }
 
+void topic_table::add_partition_to_force_reconfigure(
+  ntp_with_majority_loss entry) {
+    const auto& [it, _] = _partitions_to_force_reconfigure.try_emplace(
+      entry.ntp);
+    it->second.push_back(std::move(entry));
+    _partitions_to_force_reconfigure_revision++;
+}
+
+void topic_table::reset_partitions_to_force_reconfigure(
+  const force_recoverable_partitions_t& other) {
+    _partitions_to_force_reconfigure = other;
+    _partitions_to_force_reconfigure_revision++;
```

Comment:
why can not we use a revision in the same way as we do for other commands ?

Reply:
this is to check iterator invalidation? (not the command revision) or you mean something else?